pinot-api/src/main/java/org/apache/pinot/client/Connection.java

```diff
@@ -156,4 +157,13 @@ public ResultSetGroup get(long timeout, TimeUnit unit)
       return new ResultSetGroup(response);
     }
   }
+
+  /**
+   * connection close.
+   */
+  public void close() {
+    if(Objects.nonNull(_transport)) {
+      System.exit(0);
```

Comment:
we should never have system.exit in the code

Reply:
Yes, So I put the deleted, in JsonAsyncHttpPinotClientTransport class increased the close method.

Reply:
If you do not add the close method, the client consistently maintains the connection state, it'll experience threads hanging and resource leaks，Thanks